Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -172,11 +172,9 @@ private void RenderConfigurations()
                 RenderToggle(MRConfig.VisibleMetaFiles, "Enable visible meta files");
                 if (!MixedRealityOptimizeUtils.IsBuildTargetAndroid() && !MixedRealityOptimizeUtils.IsBuildTargetIOS() && XRSettingsUtilities.IsLegacyXRActive)
                 {
-#if UNITY_2019_3_OR_NEWER
-                    RenderToggle(MRConfig.VirtualRealitySupported, "Enable legacy XR");
```

Comment:
We might want to message this in the release notes or something, not to expect the configurator to enable XR anymore on 2019+.

Reply:
added.